CRM/Profile/Page/MultipleRecordFieldsListing.php

```diff
@@ -337,29 +339,31 @@ public function browse() {
               $editable = FALSE;
               if ($linkAction & CRM_Core_Action::UPDATE) {
                 $spec = $options[$fieldId]['attributes'];
-                switch ($spec['html_type']) {
-                  case 'Text':
-                    // Other data types like money would require some extra validation
-                    // FIXME: crmEditable currently does not support any validation rules :(
-                    $supportedDataTypes = array('Float', 'String', 'Int');
-                    $editable = in_array($spec['data_type'], $supportedDataTypes);
-                    break;
-
-                  case 'TextArea':
-                    $editable = TRUE;
-                    $fieldAttributes['data-type'] = 'textarea';
-                    break;
-
-                  case 'Radio':
-                  case 'Select':
-                  case 'Select Country':
-                  case 'Select State/Province':
-                    $editable = TRUE;
-                    $fieldAttributes['data-type'] = $spec['data_type'] == 'Boolean' ? 'boolean' : 'select';
-                    if (!$spec['is_required']) {
-                      $fieldAttributes['data-empty-option'] = ts('- none -');
-                    }
-                    break;
+                if ($spec['is_view'] == '0') {
```

Comment:
seems like this condition could be combined with the one 2 lines above to result in:

    if (!$options[$fieldId]['attributes']['is_view'] && $linkAction & CRM_Core_Action::UPDATE) {

and a much smaller patch :)

Note changing `== '0'` to `!` which imo should be more reliable when comparing to NULL, FALSE, `0` and other falsy  things.